pages/mesosphere/dcos/services/dcos-monitoring/1.3.1/operations/alertmanager/config/index.md

```diff
@@ -0,0 +1,158 @@
+---
+layout: layout.pug
+navigationTitle: Managing Alertmanager Configurations
+title: Managing Alertmanager Configurations
+menuWeight: 20
+excerpt: Enabling and configuring Alertmanager
+render: mustache
+model: ../../../data.yml
+---
+
+# Enabling Alertmanager
+
+Alertmanager is off by default.
+To enable Alertmanager, the {{ model.techName }} service must be configured to load the Alertmanager configurations from a Git repository.
+See the [Alertmanager documentation](https://prometheus.io/docs/alerting/configuration/) to learn how to create the configuration file.
+
+## Save Alertmanager configurations
+
+The Alertmanager configuration file (YAML format) must be named `config.yml`.
+You should save `config.yml` and all template files in a Git repository.
+Assume that the repository is `https://github.com/company/alertmanager-configs`.
+
+You can put `config.yml` and template files in a folder in the repository.
+For instance, `https://github.com/company/alertmanager-configs/production`.
+
+```json
+{
+  "alertmanager": {
+    "config_repository": {
+      "url": "https://github.com/company/alertmanager-configs",
+      "path": "/production"
+    }
+  }
+}
+```
+
+The Git repository can be either public or private.
+You can omit the `credentials` section if the Git repository is public.
+If the Git repository is private, you must configure the credentials to access the Git repository.
+
+## Create secrets for Git repository credentials
+
+If the Git repository containing the Alertmanager configurations is private, you must configure the secrets first.
+Currently, the following Auth types are supported.
+
+<p class="message--important"><strong>IMPORTANT: </strong>If the Git repository contains submodules, all the submodules must use the same Auth type as what's used for the repository. For example, if HTTP Auth is configured for the Git repository, all of the submodules in the repository's <code>.gitmodules</code> file must use the <code>https</code> URL scheme.</p>
```

Comment:
I see we changed "what is" to "what's" here but I don't think it improved this sentence very much.

I suggest removed "as what's" altogether -->

"If the Git repository contains submodules, all the submodules must use the same Auth type as the one used for the repository"

This same warning appears in several pages, let's make sure we fix all of them, I didn't add comments for all of the incidences.